envy.go

```diff
@@ -33,7 +33,10 @@ var env = map[string]string{}
 const GO111MODULE = "GO111MODULE"
 
 func init() {
-	Load()
+	err := Load()
+	if err != nil {
+		fmt.Printf("unable to load env file(s): %s", err.Error())
```

Comment:
Should probably add a newline at the end so it doesn't get muddled in other prints... and it might be nice to flavor/tag that text so it's easier to tell it's coming from envy.